triggermail_templates.py

```diff
@@ -128,6 +134,53 @@ def dissect_filename(self, template_filename):
         self.partner = self.settings.get("partner", self.partner) or self.partner
         self.partner = self.partner.replace("_templates", "")
 
+    def generate_pertinent_file_names_and_map(self, template_filename):
+        template_filename = template_filename.replace(self.path + '/', '')
+        image_regex = "/img/" + self.partner + "/" + "(.*?)['\"]"
+        file_map = dict()
+        all_file_names_required = [template_filename]
+        all_images_required = []
+        # go through all files this template requires
+        # we get all the filenames, their contents, and images
+        for file_name in all_file_names_required:
+            # some templates include html files that don't exist. Just because they don't exist, doesn't
+            # mean it should automatically fail the command, they are most likely in jinja "if" statements
+            try:
+                contents = read_file(os.path.join(self.path, file_name))
+            except:
```

Comment:
can you catch the specific exception, instead of all exceptions? This catch all will hide bugs. Additionally I'm not sure if it is useful for a sublime plugin, but logging the exception with a message that you are ignoring it is frequently useful when debugging